testdata/dnn/darknet/connected.cfg

```diff
@@ -0,0 +1,16 @@
+[net]
+batch=1
+channels=3
+height=4
+width=6
```

Comment:
Do you mind to choose smaller input resolution? Like `1x2x3x4`?

Reply:
okay.

Reply:
Do you want me to change input for maxpool too ?

Reply:
No, pooling looks fine for me, thanks!